packages/core/src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -127,7 +127,12 @@ export class Breadcrumbs extends AbstractPureComponent<BreadcrumbsProps> {
                     content={<Menu>{orderedItems.map(this.renderOverflowBreadcrumb)}</Menu>}
                     {...popoverProps}
                 >
-                    <span className={Classes.BREADCRUMBS_COLLAPSED} />
+                    <span
+                        aria-label="collapsed breadcrumbs"
```

Comment:
While adding an `aria-label` is a great accessibility improvement, we should avoid hardcoding the string "collapsed breadcrumbs" directly in the component. This approach doesn't support localization for international users.

Instead, can we introduce a new prop, something like `expandText`, to allow consumers of this component to provide a localized or customized string? We can set the default value of that prop to a string that best describes the action of this button.

```suggestion
                        aria-label={this.props.expandText}
```

Reply:
Are you down for a _default_ of `"collapsed breadcrumbs"`? Or do you want no `aria-label` at all?

Reply:
Can set via `overflowButtonProps`: https://github.com/palantir/blueprint/pull/6985/commits/85519823178f115b902d4572a1cbbfbd43658ac8

Reply:
> Are you down for a _default_ of `"collapsed breadcrumbs"`? Or do you want no `aria-label` at all?

Having `"collapsed breadcrumbs"` as the default seems fine to me.